src/corelibs/U2View/src/ov_sequence/AnnotatedDNAView.cpp

```diff
@@ -914,6 +914,7 @@ QString AnnotatedDNAView::addObject(GObject *o) {
     } else if (o->getGObjectType() == GObjectTypes::ANNOTATION_TABLE) {
         AnnotationTableObject *ao = qobject_cast<AnnotationTableObject *>(o);
         SAFE_POINT(NULL != ao, "Invalid annotation table!", QString::null);
+        checkAndAddObjectToOtherViews(o);
```

Comment:
Each view should work only with itself. Other views will add the object by themselves, if it is supposed to do.